src/profile/v100/entity/account/account_uniffi_fn.rs

```diff
@@ -15,6 +15,11 @@ pub fn new_account_sample_mainnet_carol() -> Account {
     Account::sample_mainnet_carol()
 }
 
+#[uniffi::export]
+pub fn new_account_sample_mainnet_diana() -> Account {
```

Comment:
new sample values with `EntityFlags` with `DeletedByUser`, ensured other sample values do not have this flag, so that AuthorizedPersonaSimple references non hidden accounts and personas...